settings.py

```diff
@@ -79,6 +79,27 @@
 # Set to True if we're allowed to use X-SENDFILE.
 XSENDFILE = False
 
+# Enable the Django Debug Toolbar for local dev.
+INSTALLED_APPS += (
+    'debug_toolbar',
```

Comment:
Nope, it'll just be always there for local dev and tests, but not on production/dev/stage, because their settings (in `sites/`) only import from `lib/settings_base.py`, not from `settings.py`.

Ideally, we'd only have this defined in the `local_settings.py` file, so it'll only be there for people who want it (and not in the tests for example), but then we're back to needing a manual step for hacking on Olympia.